unison-src/tests/cce.u

```diff
@@ -62,7 +62,8 @@ Monoid.orElse m = cases
 
 merge : (a -> a -> Boolean) -> [a] -> [a] -> [a]
 merge lte a b =
-  go acc a b = match List.at 0 a with
+  use List at
+  go acc a b = match at 0 a with
```

Comment:
Here's an example where preferring local definitions isn't necessarily a win. There's a `Remote.at` defined in this file, so just `at` would otherwise refer to `Remote.at` (instead of `List.at`). Before, TDNR was able to figure this one out. Now you need to say `List.at` if you want to refer to that in a file where you've defined something else called `at`.

Reply:
Err, the previous version called `List.at`, not `at`, which makes me not understand the comment.

Reply:
I see; so your comment was referring to the unqualified at on line 67/68, but the switch from `List.at` to `at` on line 65/66 threw me off.